### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by sanswasnthere26** [FTBTeam/FTB-App#357]
**Operating System:**
OS: Ubuntu 20.04

**Installer Version:**
Installer: 202011261605-2be2c6a678-release

**App Version:**
App: 202011261605-2be2c6a678-release

**UI Version:**
UI: 202011261605-2be2c6a678-release

**Log Files:**
Put links to your logs here
https://hastebin.com/raw/hamonuboco

**Describe the bug**
Minecraft launcher doesnt download
**To Reproduce**
Steps to reproduce the behaviour:
1. Go to  a modpack
2. Click on install
3. Scroll down to wait for it to download minecraft launcher

**Expected behaviour**
minecraft launcher should download

**Screenshots**

**Additional context**
it only seems to happen on 1.12.2 modpacks i think

**Comment by Bright-Shard** [comment]
Happens to me too. I'm on Manjaro. Is the Linux app bugged?

**Comment by Bright-Shard** [comment]
Ok, I finally got everything working. Took a while, but here's what I've found:

1. I'm pretty sure the "unable to install launcher" error is from a network error. Either your connection isn't great or some filtering software on your network is blocking the download. I had to connect to a VPN to evade network filters & get it working.
2. The next error I found is when I hit launch, the minecraft launcher wouldn't open. I had to manually install the launcher by extracting `/home/[username]/.ftba/bin/launcher.tar.gz` to fix that.
3. After that, hitting the play button from the launcher caused the game to crash. I had to set the default Java version to version 8 to get the mods to load. On Arch-based distros, run `archlinux-java` to do this. On Debian-based distros, the command should be `sudo update-alternatives --config java`, but as I use Manjaro (Which is Arch-based), I didn't test that command and can't confirm that it works.

Hopefully this helps others!
